### PR TITLE
SDESK-1412 Trim html tags in open article headline

### PR DESCRIPTION
**Pull request by pablopunk**
Removing tags on save was not a good idea because we might need those tags on the headline for styling.

I just created an angular filter to trim html tags and used it on the headline at the bottom bar.

### CODE REVIEW

File: scripts/core/filters/index.js [hunk 1]
```diff
@@ -214,4 +214,8 @@ export default angular.module('superdesk.core.filters', [])
         var cleanedValue = value || '';
 
         return cleanedValue.replace('_', ' ');
-    });
+    })
+    .filter('trimHtmlTags', () => function(value) {
```
Comment (petrjasek) [inline scripts/core/filters/index.js]: we have something defined already, maybe it should go into some `core/utils` and be used in both
https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/authoring/authoring/helpers.js#L99

Comment (pablopunk) [reply scripts/core/filters/index.js]: @petrjasek okay, I'll do that. You mean creating a new `core/utils/index.js` and exporting a `stripHtmlRaw` function that I can use in the filter as well as the tests right?

Comment (petrjasek) [reply scripts/core/filters/index.js]: yep, maybe just make it `core/utils.js` not sure if we need more now

File: scripts/apps/authoring/authoring/helpers.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import {stripHtmlTags} from '../../../core/utils';
```
Comment (petrjasek) [inline scripts/apps/authoring/authoring/helpers.js]: you can import from `core/utils`

**Pull request merged**